src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movieData: {
+    title: string,
+    description?: string,
+    imgUrl: string,
+    imdbUrl: string,
+  }
```

Comment:
```suggestion
  movieData: Film
  }
```
You have the `Film` interface. Let's use it here